terraform/ec2/amis.tf

```diff
@@ -150,6 +167,14 @@ variable "amis" {
       family = "linux"
       login_user = "ec2-user"
       arch = "amd64"
+      user_data = <<EOF
```

Comment:
terraform does not allow you to define string constant that can be used in variable defaults, and you can use local or variable in variable defaults ... /w